Configure

```diff
@@ -24,6 +24,14 @@ use OpenSSL::config;
 
 # see INSTALL.md for instructions.
 
+# Set this variable to 1 if you want to skip all the Engine related tests
+# Re-enable with 0. (and you also should add back 'afalgeng' in
+# Configurations/10-main.conf to the enable array at (possible) line 697
+# (targets/linux_generic32)
+# This is needed for engine removal process, because we can't do it in one big
```

Comment:
this sounds awkward. why we just postpone the removal of 'afalgemg' option to very last commit on this branch. I admit I must be missing great deal of information here.

I'm fine with having  a switch which keeps engine tests disabled (skipped) and if one needs it flips the switch to enable those tests. but editing additional file when enabling switch feels odd.

Reply:
It does not have to be the last commit, it's only that we try to separate out to make small PR's (not like this one https://github.com/openssl/openssl/pull/28384). It is better to review and also we can share the load of removing engines among more people

Reply:
Splitting to reasonably sized PRs helps here already (as I am just trying to do some parts in parallel), it cannot be done as one PR anymore.

Currently the major block is tests failing (missing this PR fix). Could we just do it as suggested for now so the parallel PRs can be actually merged for the feature branch?